modules/images/webp-uploads/helper.php

```diff
@@ -20,6 +20,14 @@ function webp_uploads_get_upload_image_mime_transforms() {
 		'image/webp' => array( 'image/webp', 'image/jpeg' ),
 	);
 
+	// Check Setting for generating mimes.
+	if ( true === (bool) get_option( 'generate_webp_and_jpeg' ) ) {
```

Comment:
@mukeshpanchal27 aren't we checking for the default setting value here if the site hasn't saved settings yet? I guess it should be `1` as the default value.

Reply:
Now that we think of it, it seems default value for the setting should be `0` -> false to avoid generating duplicate mime images.